public/coffee/ide/editor/directives/aceEditor/auto-complete/AutoCompleteManager.coffee

```diff
@@ -89,18 +121,33 @@ define [
 		onChange: (change) ->
```

Comment:
can you move this labels specific code into LabelsManager so that module is self contained
I think it can work just like the SpellCheckManager.